dhall/tests/Dhall/Test/QuickCheck.hs

```diff
@@ -383,7 +373,7 @@ instance (Arbitrary s, Arbitrary a) => Arbitrary (Expr s a) where
             % (1 :: W "With")
             % (0 :: W "Note")
             % (7 :: W "ImportAlt")
-            % (7 :: W "Embed")
+            % (0 :: W "Embed")
```

Comment:
This is a temporary workaround so we don't run into https://github.com/dhall-lang/dhall-haskell/issues/1925.

A better workaround would be to filter the `Expr`s used in `idempotenceTest` to ensure that any `Embed` has a `Nothing` `header`.